query-node/mappings/src/workingGroups.ts

```diff
@@ -671,6 +675,42 @@ export async function workingGroups_StatusTextChanged({ store, event }: EventCon
   await store.save<StatusTextChangedEvent>(statusTextChangedEvent)
 }
 
+export async function workingGroups_LeadRemarked({ store, event }: EventContext & StoreContext): Promise<void> {
+  const [metadataByte] = new WorkingGroups.LeadRemarkedEvent(event).params
+  const group = await getWorkingGroup(store, event)
+
+  const metadata = deserializeMetadata(RemarkMetadataAction, metadataByte)
+  if (metadata?.moderatePost) {
+    if (group.name !== 'forumWorkingGroup') {
+      return invalidMetadata(`The ${group.name} is incompatible with the remarked moderatePost`)
+    }
+    const { postId, rationale } = metadata.moderatePost
+    const actor = await getWorkingGroupLead(store, group.name)
+
+    await moderatePost(store, event, 'leadRemark', postId, actor, rationale)
```

Comment:
if a lead provides an invalid postId and moderatePost throws, will this not kill the processor? Since the postId is not validated by the runtime because it is in the metadata..